terraform/config.tf

```diff
@@ -77,3 +84,11 @@ provider "aws" {
 provider "fastly" {
   api_key = var.credentials["fastly"]
 }
+
+provider "sigsci" {
+  alias          = "firewall"
+  corp           = "python"
+  email          = "infrastructure-staff@python.org"
```

Comment:
Should this be `var.ngwaf_email` instead?

Reply:
That var was removed and did not exist in this scope but [3e3f1ed](https://github.com/pypi/infra/pull/172/commits/3e3f1edff661c578b04b1bd67562e90ca9651022) cleans that up